pkg/infoschema/perfschema/init.go

```diff
@@ -62,6 +64,7 @@ func Init() {
 				c.ID = int64(i) + 1
 			}
 			meta.DBID = dbID
+			meta.State = model.StatePublic
```

Comment:
I think the master code lost this line to set the table state to public... Though it still works well without this state, I still fix it to make the meta complete.